src/components/Drawer/Drawer.Footer.tsx

```diff
@@ -16,14 +16,24 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { ReactElement, ReactNode } from 'react'
+import React, { ReactElement } from 'react'
 
-export type DrawerFooter = ReactNode
+import { DrawerFooter, FooterProps } from './Drawer.types'
+import styles from './Drawer.module.css'
 
-export type FooterProps = {
-  footer: DrawerFooter,
-}
+export const Footer = ({ footer }: FooterProps): ReactElement | null => {
+  if (React.isValidElement(footer)) {
+    return footer
+  }
 
-export const Footer = ({ footer }: FooterProps): ReactElement => {
-  return <footer>{footer}</footer>
+  const drawerFooter = footer as DrawerFooter
```

Comment:
let's make a isDrawerFooter that returns the proper type casting instead of using `as`

Reply:
I don't think there is a clean way to do that, I personally prefer to do it this way

Reply:
done